### DISCUSSION TITLE
Adicionar como instalar o ambiente de desenvolvimento no CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by george-gca** [levxyca/diciotech#193] [Ideas]
Se uma pessoa quiser contribuir com código que não sejam novos termos, ela precisa primeiro instalar o ambiente de desenvolvimento na sua máquina para então poder fazer testes locais antes de submeter uma PR. Seria útil ter uma seção dentro do arquivo CONTRIBUTING.md ensinando como instalar as dependências necessárias.

**Comment by levxyca** [comment]
@george-gca perfeito! adicionei aqui algo bem básico, mas podemos ir melhorando com o tempo https://github.com/levxyca/diciotech/blob/main/CONTRIBUTING.md#%EF%B8%8F-instalando-ambiente-de-desenvolvimento-para-contribuir

**Reply by george-gca** [reply]
Mas não acho que o servidor local seja opcional. Só abrindo o index.html não carrega o javascript e consequentemente os termos. Acho que vale dizer como executar um servidor http localmente. Eu conheço um que [tem que instalar via npm](https://www.npmjs.com/package/http-server), mas eu uso o que vem com o [python](https://realpython.com/python-http-server/#how-to-start-pythons-httpserver-in-the-command-line).

**Reply by george-gca** [reply]
E eu botaria o sass como opcional, se a pessoa não for alterar nada visualmente não é necessário. Eu estava tentando brincar rapidinho com a busca por exemplo, então não preciso.

**Reply by levxyca** [reply]
Ótimas observações @george-gca! Obrigada! Não vou conseguir fazer esses updates por agora, então vou criar uma issue pra atualizarmos isso 🫰🏻

**Reply by levxyca** [reply]
@george-gca criei a issue em https://github.com/levxyca/diciotech/issues/195 🚀 acho que podemos fechar essa discussion, o que acha?
